aws/asg_test.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"reflect"
 	"testing"
+	"os"
```

Comment:
This continues to baffle me. I clearly need this import, why is it suggesting I remove it?

Reply:
Just ignore the bot, if it does not makes sense, like here.
Maybe it'S because of this:
```
$ make lint
golangci-lint run ./...
aws/asg_test.go:4: File is not `goimports`-ed (goimports)
	"fmt"
Makefile:23: recipe for target 'lint' failed
make: *** [lint] Error 1
The command "make lint" exited with 2.
```